src/node_crypto.cc

```diff
@@ -4331,6 +4371,8 @@ SignBase::Error Verify::VerifyFinal(const char* key_pem,
   int r = 0;
   EVP_PKEY_CTX* pkctx = nullptr;
 
+  ERR_set_mark();
+
   bp = BIO_new_mem_buf(const_cast<char*>(key_pem), key_pem_len);
```

Comment:
setting the err mark. https://github.com/nodejs/node/issues/5444#issuecomment-226419786. Im not sure where else we might need to do this in this file.

Reply:
I admit I don't understand why that is necessary when you have a `ClearErrorOnReturn` in the binding function.